_maps/map_files/MetaStation/MetaStation.v41G.dmm

```diff
@@ -4017,7 +4017,7 @@
 "bzm" = (/obj/machinery/atmospherics/pipe/manifold/supply/hidden{dir = 8},/turf/simulated/wall/r_wall,/area/atmos)
 "bzn" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/wall/r_wall,/area/atmos)
 "bzo" = (/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 10},/turf/simulated/wall/r_wall,/area/atmos)
-"bzp" = (/obj/machinery/computer/telecomms/traffic{network = "tcommsat"},/turf/simulated/floor/plasteel{icon_state = "grimy"},/area/tcommsat/computer{name = "\improper Telecoms Control Room"})
+"bzp" = (/obj/structure/table/wood/poker,/obj/effect/spawner/lootdrop{loot = list(/obj/item/weapon/gun/projectile/revolver/russian = 5, /obj/item/weapon/storage/box/throwing_stars, /obj/item/toy/cards/deck/syndicate = 2); name = "gambling valuables spawner"},/turf/simulated/floor/wood,/area/crew_quarters/bar)
```

Comment:
He's added a poker table with a lootdrop including the (possibility of) russian revolvers, syndiecards, and throwing stars where the main console used to be.